ts/de_DE.ts

```diff
@@ -317,7 +317,7 @@
     <message>
         <location filename="gui.py" line="1268"/>
         <source>In what units you want to scale?</source>
-        <translation>In welcher Einheit möchtest du skalieren?</translation>
+        <translation>In welcher Einheit möchten Sie skalieren?</translation>
```

Comment:
Most of the times in 3D Modelling the environment is called "Szene" anyway in German too so i just went with that. I guess "Umgebung" would make sense too, but I'm very vary to differ tooo much from a 1:1 translation because it could make understanding cross-language harder. If it's called scene in english and Szene in german then it's pretty easy to associate.